spec/github_pages_health_check_domain_spec.rb

```diff
@@ -474,4 +474,89 @@ def mx_packet
       expect(domain_check.dns?).to be(false)
     end
   end
+
+  context "https" do
+    let(:domain) { "pages.github.com" }
+    let(:domain_check) { make_domain_check(domain) }
+
+    context "a site that supports HTTPS" do
+      let(:domain) { "pages.github.com" }
+
+      before do
+        stub_request(:head, "https://pages.github.com/").
+           to_return(:status => 200, :headers => {:server => "GitHub.com"})
+         allow(domain_check.send(:https_response)).to receive(:return_code) { :ok }
```

Comment:
We're using Webmock the way it's describes, and `return_code` is where the SSL errors are exposed, but looking at the raw object, it seems Webmock doesn't stub the `response_code`.